### PR TITLE
Modified the UI of events log and switch tab

### LINKED ISSUE #8286: Modify the UI of events log switch tab
**Describe the bug**

Currently the switch tab of events & daily road and filters button in the consultation page are utilizing two separate rows, unnecessarily 

**To Reproduce**

1. Go to the patient tab
2. click on a active consultation
3. See error

**Expected behavior**

- [ ] keep the daily round & events and filter button to a single row 
- [ ] Remove the sort-by button in the events green border to make it similar to the filter button in daily rounds
- [ ] When the sort by is active in events, make it the primary color to show sort by is active 

**Screenshots**

image
image

### PR DESCRIPTION
**Pull request by JOSHIK27**
## Proposed Changes

- Fixes #8286 
- Erased the logic for both sort buttons which were initially present in child components i.e ([eventlist](src/Components/Facility/ConsultationDetails/Events/EventsList.tsx) and [dailroundslist](src/Components/Facility/Consultations/DailyRoundsList.tsx)) and brought the logic to parent component which was rendering child components.
- In the [parent](src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx) component, 2 query state variables for both sort buttons were created and passed them as props to child components. And DailyRoundsSortDropdown (which is popover) component is created in the parent component.
- [DailyRoundsSortDropdown](src/Components/Facility/Consultations/DailyRoundsFilter.tsx) styling was fixed as we just need to display filter icon (not text)
- For eventlist sort button, a new popover was created to make it look exactly same as DailyRoundsSortDropdown popover.

https://github.com/user-attachments/assets/8e8cb2e8-777d-4a60-8327-dbe4e6d62e24

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Please follow the PR template and checklist.

**Comment by JOSHIK27** [comment]
> Please follow the PR template and checklist.

I have added proposed changed in the pr template. can you check if everything is ok or if anything needs to changed @rithviknishad ?

File: src/pluginMap.ts
Comment (rithviknishad) [inline src/pluginMap.ts:1]: this file is gitignored. why was it checked in?

**Review by rithviknishad — CHANGES_REQUESTED** [review]
lgtm, just a minor change

File: src/pluginMap.ts
Comment (JOSHIK27) [reply src/pluginMap.ts:1]: When i merged develop branch with my feature branch this was in unstaged files of git. I added it to staging area and then pushed it. Thats surprising.

The feature branch had this file for very long time, when i merged it with develop this was added to staging area. It was my mistake. Shall i delete this file ?

Comment (rithviknishad) [reply src/pluginMap.ts:1]: yup

Comment (JOSHIK27) [reply src/pluginMap.ts:1]: > yup

Done @rithviknishad

**Review by rithviknishad — APPROVED** [review]

**Comment by bodhish** [comment]
@JOSHIK27 the `Merge Checklist` is still not completed. Please complete it. 

- [ ]  Ensure that UI text is kept in I18n files.
- [ ]  Prep screenshot or demo video for changelog entry, and attach it to issue.

**Comment by JOSHIK27** [comment]
> @JOSHIK27 the `Merge Checklist` is still not completed. Please complete it.
> 
>     * [ ]   Ensure that UI text is kept in I18n files.
> 
>     * [ ]   Prep screenshot or demo video for changelog entry, and attach it to issue.

Attached demo video above. And the pr above does not introduce any text to the ui (just layout fixes). Please let me if anything is needed. Thanks.

File: src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx [hunk 1]
```diff
@@ -651,32 +667,107 @@ export const ConsultationUpdatesTab = (props: ConsultationTabProps) => {
...
-            currentTab={showEvents ? 1 : 0}
-          />
+          <div className="flex items-center">
+            <Tabs
+              className="mr-2 mt-3 w-full lg:w-full"
+              tabs={[
+                {
+                  text: (
+                    <div className="flex items-center justify-center gap-1 text-sm">
+                      Events
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx]: missing i18n

File: src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx [hunk 2]
```diff
@@ -651,32 +667,107 @@ export const ConsultationUpdatesTab = (props: ConsultationTabProps) => {
...
+                      <span className="rounded-lg bg-warning-400 p-px px-1 text-xs text-white">
+                        beta
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx]: missing i18n

File: src/Components/Facility/Consultations/DailyRoundsFilter.tsx [hunk 3]
```diff
@@ -42,84 +42,81 @@ export default function DailyRoundsFilter(props: Props) {
...
+                {...field("rounds_type")}
+                label={t("LOG_UPDATE_FIELD_LABEL__rounds_type")}
+                options={DailyRoundTypes}
+                placeholder={t("show_all")}
+                optionLabel={(o) => t(`ROUNDS_TYPE__${o}`)}
+                optionValue={(o) => o}
+              />
+              <TextFormField
+                {...field("taken_at_after")}
+                label="Measured after"
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/DailyRoundsFilter.tsx]: missing i18n

**Review by rithviknishad — COMMENTED** [review]
a self review of the code changes would help identify missing translations

File: src/Locale/en.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
   "APPETITE__SATISFACTORY": "Satisfactory",
   "AUTOMATED": "Automated",
   "BED_WITH_OXYGEN_SUPPORT": "Bed with Oxygen Support",
+  "BETA": "beta",
```
Comment (bodhish) [inline src/Locale/en.json]: use small case for all I18n strings. (Only make changes to the onces that you have introduced.)

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Locale/en.json (hunk 4 above)
Comment (JOSHIK27) [reply src/Locale/en.json]: Sure making the changes.

**Review by rithviknishad — APPROVED** [review]

**Review by bodhish — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
image

- [ ] add a spacing between the card and the switch tabs

image

- [ ] When a user applies any sort-by option, the filter icon button should change to the primary color, the same as the behavior in the daily round filter where the button turns primary when a filter is active.

**Comment by JOSHIK27** [comment]
> image
> 
>     * [ ]  add a spacing between the card and the switch tabs
> 
> 
> image
> 
>     * [ ]  When a user applies any sort-by option, the filter icon button should change to the primary color, the same as the behavior in the daily round filter where the button turns primary when a filter is active.

Please let me know if anything needs to be changed.

https://github.com/user-attachments/assets/ff07645c-da99-4d92-9ca1-3946250f6f0b

**Comment by nihal467** [comment]
LGTM

**Pull request merged**